spark/dl/src/test/scala/com/intel/analytics/bigdl/integration/torch/LSTMPeepholeSpec.scala

```diff
@@ -668,6 +668,7 @@ class LSTMPeepholeSpec  extends TorchRNNSpec {
         assert(abs(v1 - v2) <= 1e-8)
         v1
       })
+    case _=>
```

Comment:
maybe you can delete `case ((key: Int, value: Tensor[_])) =>`, just use `asInstanceOf`